CharlieBackend.Api/Middlewares/IsAccountActiveMiddleware.cs

```diff
@@ -59,32 +51,24 @@ public async Task InvokeAsync(HttpContext context, IAccountService accountServic
 
                     if ((bool)!isActive)
```

Comment:
```suggestion
                    if (!isActive)
```

Reply:
It turned out that explicit `bool` cast is required since `isActive` is of `Nullable<bool>` type.